bitnami/opensearch/values.yaml

```diff
@@ -3405,3 +3405,181 @@ dashboards:
     ## @param dashboards.persistence.dataSource Custom PVC data source
     ##
     dataSource: {}
+
+## @section OpenSearch Snapshots Parameters
+snapshots:
+  ## @param snapshots.command Override default container command (useful when using custom images)
+  ##
+  command:
+    - /usr/bin/curl
```

Comment:
Is it necessary to add this here? I was thinking about writing the command directly in the template file

Reply:
Not necessary at all, having it hard-coded in the template should work just as well for our purposes. Just that this lets the user customize the command, should they need to. We already let them change image, and if the do that it would very likely be required to change the command also (e.g. because `curl` exists in another path in their custom image).

There are `command` overrides in other parts of the `values.yaml` file too, the only difference here is that I included the default value as part of the `values.yaml` file instead of using logic in the template to check for custom values and then conditionally changing the command.

No rights or wrongs here, really. Just preference. But I do think that we need to accommodate for custom `command` if we want to let users provide custom images.

What do you think?